src/azure-cli/azure/cli/command_modules/policyinsights/commands.py

```diff
@@ -36,6 +36,8 @@ def load_command_table(self, _):
     with self.command_group('policy state', policy_states_sdk, client_factory=policy_states_operations) as g:
         g.custom_command('list', 'list_policy_states')
         g.custom_command('summarize', 'summarize_policy_states')
+        g.custom_command('trigger-scan', 'trigger_policy_scan',
```

Comment:
Because you support --no-wait here, I will recommend you to add `wait` command here using `g.wait_command` to query status.

Reply:
@Juliehzl, This is a POST action using the 202 + location header async pattern.  Without the URL from the location header in the original response there is no way to wait after the fact.  I can't find any existing instances of a command attempting to expose this URL and pass it into a custom_wait_command.  

Do you recommend going down that path (i.e. returning a polling URL from trigger-scan if --no-wait is used)?  I can also remove --no-wait altogether (although this command can take minutes) or not include a 'wait' command in this instance.

Reply:
@Juliehzl whats more aggreable to you?  Return a URI form the trigger command that is passed to a custom wait command or not including the wait command for async post actions (+ potentially removing --no-wait)?

Reply:
@Juliehzl need some input as either solution is a special case

Reply:
I see. `Provision State` is not possible to query your operation status, right?

Reply:
Correct.  ProvisioningState and polling on the original resource is only used for async PUT calls (i.e. resource creation or updates).